addons/project/migrations/7.0.1.1/post-migration.py

```diff
@@ -65,6 +65,27 @@ def createProjectAliases(cr, pool):
             """, (alias_id, id))
 
 
+def setStageFromState(cr):
```

Comment:
From pep8: Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.
That would be set_stage_from_state()